bin/postman-test/collections/core-metadata.postman_collection.json

```diff
@@ -4962,7 +4962,7 @@
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "{\r\n  \"origin\":1471806386919,\r\n  \"name\":\"TestProfileToPostUniqe\",\r\n  \"description\":\"BACnet Honeywell thermostats\",\r\n  \"manufacturer\":\"Honeywell\",\r\n  \"model\":\"ABC123\",\r\n  \"labels\":[\"bacnet\",\"thermostat\",\"Honeywell\"],\r\n  \"commands\":[\r\n    {\r\n      \"origin\":1471806386919,\r\n      \"name\":\"cooling point\",\r\n      \"get\":\r\n        {\r\n          \"path\":\"/cooling\",\r\n          \"response\":{\r\n            \"code\":\"200\",\r\n            \"errorDescription\":\"not a float\",\r\n            \"expectedValues\":\"temperature\"\r\n          }\r\n        },\r\n      \"put\":{\r\n        \"path\":\"/cooling\",\r\n        \"response\":{\r\n          \"code\":\"200\",\r\n          \"errorDescription\":\"not a float\",\r\n          \"expectedValues\":\"temperature\"\r\n        },\r\n        \"parameters\":[  \r\n               {  \r\n                  \"name\":\"coolingpoint\",\r\n                  \"value\":\"float\",\r\n                  \"uom\":\"string\"\r\n               }\r\n        ]\r\n      }\r\n    }\r\n  ],\r\n  \"objects\":{\r\n    \"resources\":{\r\n      \"CurrentHumidity\":{\r\n        \"get\":[\r\n          {\r\n            \"parameter\":\"CurrentHumidity\",\r\n            \"property\":\"presentValue\",\r\n            \"operation\":\"get\",\r\n            \"object\":\"AnalogValue_22\"\r\n          }\r\n        ]\r\n      }\r\n    }\r\n  }\r\n}"
+							"raw": "{\r\n  \"origin\":1471806386919,\r\n  \"name\":\"TestProfileToPostUniqe\",\r\n  \"description\":\"BACnet Honeywell thermostats\",\r\n  \"manufacturer\":\"Honeywell\",\r\n  \"model\":\"ABC123\",\r\n  \"labels\":[\"bacnet\",\"thermostat\",\"Honeywell\"],\r\n  \"commands\":[\r\n    {\r\n      \"origin\":1471806386919,\r\n      \"name\":\"cooling point\",\r\n      \"get\":\r\n        {\r\n          \"path\":\"/cooling\",\r\n          \"response\":{\r\n            \"code\":\"200\",\r\n            \"errorDescription\":\"not a float\",\r\n            \"expectedValues\":\"temperature\"\r\n          }\r\n        },\r\n      \"put\":{\r\n        \"path\":\"/cooling\",\r\n        \"response\":{\r\n          \"code\":\"200\",\r\n          \"errorDescription\":\"not a float\",\r\n          \"expectedValues\":\"temperature\"\r\n        },\r\n        \"parameters\":[  \r\n               {  \r\n                  \"name\":\"coolingpoint\",\r\n                  \"value\":\"float\",\r\n                  \"uom\":\"string\"\r\n               }\r\n        ]\r\n      }\r\n    }\r\n  ],\r\n  \"resources\":[{\r\n    \"get\":[\r\n      {\r\n        \"parameter\":\"CurrentHumidity\",\r\n        \"property\":\"presentValue\",\r\n        \"operation\":\"get\",\r\n        \"object\":\"AnalogValue_22\"\r\n      }\r\n    ]\r\n  ]}\r\n}"
```

Comment:
There's a JSON format error at the very end of this line. Should end as follows:

`\"operation\":\"get\",\r\n        \"object\":\"AnalogValue_22\"\r\n      }\r\n    ]\r\n  }]\r\n}"`

Notice the final ]} is reversed in the PR.

Reply:
Good catch! I had a kind of dyslexia :)